frontend/lib/resolution/call-init-deinit.cpp

```diff
@@ -514,6 +514,14 @@ void CallInitDeinit::resolveAssign(const AstNode* ast,
     return;
   }
 
+  if (auto call = ast->toOpCall()) {
+    if (call->op() == "=" && call->child(0)->isTuple()) {
+      // Tuple unpacking assignment, handled directly by Resolver
+      // (a, b, c) = foo();
+      return;
+    }
+  }
```

Comment:
Will we still perform the call init-deinit logic elementwise somewhere? It seems like although we don't need to resolve `=` anymore, `CallInitDeinit` does more than just that.

Reply:
I think ``CallInitDeinit::handleAssign`` will process the mentions of variables for this purpose.